.github/CODEOWNERS

```diff
@@ -1 +1 @@
-* @holgiosalos @valeryibarra @aperdomob @manuelq12 @JuanDelahozMiranda
\ No newline at end of file
+@holgiosalos @valeryibarra @aperdomob @manuelq12 @JuanDelahozMiranda
```

Comment:
Here you need to add the: * symbol at the begining and also add an End Of File(line break)

Reply:
You can do this to add the EOF line automatically [Method I](https://stackoverflow.com/questions/44704968/visual-studio-code-insert-newline-at-the-end-of-files).